ecosystem/gstreamer_plugin/gst_mtl_st20p_rx.c

```diff
@@ -433,25 +452,43 @@ static gboolean gst_mtl_st20p_rx_negotiate(GstBaseSrc* basesrc) {
   return TRUE;
 }
 
-static GstFlowReturn gst_mtl_st20p_rx_create(GstBaseSrc* basesrc, guint64 offset,
-                                             guint length, GstBuffer** buffer) {
-  GstBuffer* buf;
-  Gst_Mtl_St20p_Rx* src = GST_MTL_ST20P_RX(basesrc);
+static GstFlowReturn gst_mtl_st20p_rx_get_external_frame(Gst_Mtl_St20p_Rx* src,
+                                                         GstBuffer** buffer) {
+  struct st_frame* frame;
+  for (int i = 0; i < src->retry_frame; i++) {
```

Comment:
```suggestion
  struct st_frame* frame;
  
  for (int i = 0; i < src->retry_frame; i++) {
```

Reply:
Fixed in bedd0895579ff54984a2df2d2aed7191e446539a